src/env/prod/terraform.tfvars

```diff
@@ -118,7 +118,7 @@ mock_ec_enabled  = false
 mock_psp_enabled = false
 
 # api_config
-api_config_enabled = false
+api_config_enabled = true
```

Comment:
now that api-config is enabled in all environments is `api_config_enabled` variable still needed?

Reply:
variable removed